Header 	header
string 	pipe
string 	version
uint32 	frame

TobiIcClassifier[] IcClassifiers
